TestClient/index-template.html

```diff
@@ -109,7 +106,7 @@ <h2>Encrypted Data JSON</h2>
     //section Config.js
     const appsettings = {
       clientId: '__FRONTEND_CLIENT_ID__',
-      authority: 'https://login.microsoftonline.com/common',
+      authority: '__AUTHORITY_URI__/common',
```

Comment:
Is this still correct to have /common? Or should it be updated to use the tenantId?